mapbox/libjava-core/src/main/java/com/mapbox/services/commons/models/Position.java

```diff
@@ -31,15 +31,15 @@ private Position(double longitude, double latitude, double altitude) {
     this.latitude = latitude;
     this.altitude = altitude;
 
-    if (latitude < -90 || latitude > 90) {
+    if (latitude < -90 || latitude > 90 && latitude != Double.POSITIVE_INFINITY) {
```

Comment:
Let's do it a bit more readable with `latitude != Double.POSITIVE_INFINITY && (latitude < -90 || latitude > 90)`